docs/understanding-radicle/how-radicle-works.md

```diff
@@ -0,0 +1,316 @@
+---
+id: how-radicle-works
+title: How Radicle works
+---
+
+import Highlight from '@site/src/components/Highlight'
+
+At its most global, Radicle is a decentralized autonomous organization (DAO) solving challenges around sovereign code
+infrastructure and incentives for open-source work.
+
+If you're curious about that global vision, we recommend reading [_Towards decentralized code
+collaboration_](https://radicle.mirror.xyz/Qz4Rr0R54KLPurqqswsQ2tfRtBs9MOYpw3dlY9VjyQ8) and [_Introducing
+RAD_](https://radicle.mirror.xyz/CgcHpSXUlPvwMVaUVVaJ7r8bIJI2BOKOytaI9-nO9oY), which details the governance token used
+by the DAO.
+
+However, for the sake of this document, we'll focus on the user-facing components of this ecosystem &mdash; what you can
+interact with, as a user, to securely collaborate on software over a peer-to-peer network built on Git.
+
+## Key components of the Radicle ecosystem
+
+There are a few important open-source protocols, concepts, projects, and assets that integrate to make [Radicle's
+vision](https://radicle.mirror.xyz/Qz4Rr0R54KLPurqqswsQ2tfRtBs9MOYpw3dlY9VjyQ8) for decentralized code collaboration
+possible.
+
+- [**Identity**](#radicle-identity): A cryptographically-signed JSON document that identifies a device or a project,
+  stored alongside standard Git data when hosted, cloned, and replicated.
+- [**Network**](#radicle-network): The people who use Radicle tooling and the projects they collaborate on,
+  which is supported by seed nodes, which provide a data availability layer through a monorepo of projects and
+  identities.
+- [**Peer-to-peer protocol**](#peer-to-peer-protocol): The protocol for disseminating Git repositories via gossip-based
+  replication, enabling the hosting and sharing of projects without knowing where it's physically stored within the
+  Radicle network.
+- [**Command-line tooling (`rad`)**](#command-line-tooling-rad): The core tool for managing your code, project, and
+  identity.
+- [**Web app**](#web-app): The web-based interface for discovering, viewing, and sharing Radicle projects.
+- [**Open-source funding (Drips)**](#open-source-funding-drips): An Ethereum protocol for continuously stream funds to
+  others, which will be integrated into the [web app](#web-app) as new value flows for developers to sustain their FOSS
+  work.
+
+## Radicle identity
+
+Everything Radicle is trying to build around decentralized code collaboration starts with the concept of identity. In
+Radicle, your identity is a *portable* JSON document that defines a user or a project on the Radicle network.
+
+The most important part is the hashed content identifier—the **peer ID**—which identifies it uniquely wherever it's
```

Comment:
I think this is not quite correct. The content identifier (URN) is not the same as the Peer ID (which designates a device). Maybe @cloudhead can jump in to clarify this?

Reply:
Yeah. Peer ID represents a device.

Reply:
To elaborate, the Peer ID is a hashed Ed25519 public key. This is the cryptographic identifier for a device and is tied to the signing key that the person uses to attest to their actions within the protocol. Others than can then verify these actions using the public key.